pkg/etcd/etcd.go

```diff
@@ -65,7 +70,18 @@ type GetterSetter interface {
 // Returns:
 // 	This puts an *etcd.Client into the context.
 func CreateClient(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
-	url := p.Get("url", "http://localhost:4001").(string)
+	host := os.Getenv(hostEnvVar)
+	port, err := strconv.Atoi(os.Getenv(portEnvVar))
```

Comment:
`p.Get(...)` returns an empty string when run on my cluster, so I put this in as a quick & dirty fix. I will revisit when things are stable